### ISSUE TITLE
Set Permission vagueness

### ISSUE THREAD

**Issue by annevk** [w3c/permissions#386]
It's not at all apparent what this means:

> Interpret _parameters_.state as if it were the result of an invocation ...

Presumably this is meant to end up changing the permission, but how?

cc @johannhof

**Comment by jyasskin** [comment]
Yes, by interacting with https://www.w3.org/TR/permissions/#dfn-permission-state step 5:
> If there was a previous invocation of this algorithm with the same descriptor and settings, returning previousResult, and the user agent has not received [new information about the user's intent](https://www.w3.org/TR/permissions/#dfn-new-information-about-the-user-s-intent) since that invocation, return previousResult.

If you're migrating all this to a permission store anyway, you can do it the more straightforward way.
